scripts/gatkcondaenv.yml

```diff
@@ -34,9 +34,12 @@ dependencies:
   - protobuf==3.5.0.post1
   - pymc3==3.1
   - pyparsing==2.2.0
+  - pysam==0.13
   - python-dateutil==2.6.1
   - pytz==2017.3
+  - pyvcf==0.6.8
   - pyyaml==3.12
+  - scikit-learn==0.19.1
```

Comment:
scikit-learn might be worth keeping, if it doesn't add to the Docker size too much.  However, I think avoiding dependencies like pysam and pyvcf was definitely part of the original deal.

Reply:
I definitely want to remove pysam, pyvcf, and biopython ASAP, but for completeness we wanted to release the tool with training as an option.  Can we let them in temporarily? Should we split the docker into one that has everything one that is slimmer?  What's the easiest way to see how much each dependency costs us in docker size?

Reply:
They're fine as a short term stopgap for this PR (and scikit-learn should be fine if we need it for CNV). But we should make a ticket to make sure the code and dependencies for the others get removed as soon as possible, since we really don't want to acquire code that depends on them. For crude approximation of the docker size delta, I usually just compare the size reported in the travis build log with a recent build (the second one is this PR - not too bad though biopython isn't included in this):

<img width="817" alt="screen shot 2018-02-08 at 8 02 01 am" src="https://user-images.githubusercontent.com/10062863/35974302-8e30ba80-0ca6-11e8-95ec-5f228029ebef.png">
<img width="810" alt="screen shot 2018-02-08 at 8 06 34 am" src="https://user-images.githubusercontent.com/10062863/35974421-0781ba92-0ca7-11e8-9193-9427250083f6.png">